python/spec/dependabot/python/update_checker/pipenv_version_resolver_spec.rb

```diff
@@ -255,8 +255,8 @@
                 to start_with("Dependabot detected the following Python")
               expect(error.message).to include("3.4.*")
               expect(error.message).
-                to include("supported in Dependabot: 3.8.4, 3.8.3, 3.8.2, "\
-                           "3.8.1, 3.8.0")
+                to include("supported in Dependabot: 3.8.5, 3.8.4, 3.8.3, "\
+                           "3.8.2, 3.8.1, 3.8.0")
```

Comment:
Only really need to check the first few 
```suggestion
                to include("supported in Dependabot: 3.8.5, 3.8.4, 3.8.3")
```

Reply:
Isn't it a bit misinforming?

Reply:
Yeah find to keep as is but we're already only testing the start of the list